src/main/java/org/folio/innreach/domain/service/impl/ContributionValidationServiceImpl.java

```diff
@@ -125,6 +130,13 @@ private boolean isExcludedStatisticalCode(UUID centralServerId, List<UUID> stati
     return DO_NOT_CONTRIBUTE_CODE.equals(suppressionCode);
   }
 
+  /*if Item's effective location is matched with Contribution Criteria's excluded locations*/
```

Comment:
I would clean up this comment a bit. Prefer `//` for single line comments and capitalize first letter

Reply:
Done